lib/discourse_chat/provider/teams/teams_provider.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module DiscourseChat::Provider::TeamsProvider
+  PROVIDER_NAME = "teams".freeze
+  PROVIDER_ENABLED_SETTING = :chat_integration_teams_enabled
+  CHANNEL_PARAMETERS = []
+
+  def self.trigger_notification(post, channel, rule)
+    display_name = "@#{post.user.username}"
+    full_name = post.user.name || ""
+
+    topic = post.topic
+
+    category = ''
+    if topic.category&.uncategorized?
+      category = "[#{I18n.t('uncategorized_category_name')}]"
+    elsif topic.category
+      category = (topic.category.parent_category) ? "[#{topic.category.parent_category.name}/#{topic.category.name}]" : "[#{topic.category.name}]"
+    end
+
+    message = {
+      "@type": "MessageCard",
+      "summary": topic.title,
+      "sections": [{
+        "activityTitle": "[#{topic.title} #{category} #{topic.tags.present? ? topic.tags.map(&:name).join(', ') : ''}](#{post.full_url})",
+        "activitySubtitle": post.excerpt(SiteSetting.chat_integration_rocketchat_excerpt_length, text_entities: true, strip_links: true, remap_emoji: true),
```

Comment:
Need to replace the reference to rocketchat here

Reply:
Hahaha sure😉